opds.py

```diff
@@ -62,6 +62,10 @@ def default_lane_url(cls):
     def top_level_title(self):
         return "All Books"
 
+    def facet_url(self, facets):
```

Comment:
In the feeds I've seen, they're just regular feed urls with the proper parameters and some extra attributes to signify their OPDS facet status:

```xml
  <link opds:activeFacet="true" href="https://circulation.librarysimplified.org/feed/eng/Adult%20Fiction?available=all&amp;collection=main&amp;order=author" opds:facetGroup="Sort by" rel="http://opds-spec.org/facet" title="Author"/>
  <link href="https://circulation.librarysimplified.org/feed/eng/Adult%20Fiction?available=all&amp;collection=main&amp;order=title" opds:facetGroup="Sort by" rel="http://opds-spec.org/facet" title="Title"/>
```